messages/mcu_messaging_main.proto

```diff
@@ -796,3 +797,12 @@ message BatteryResetReason
 }
 
 message ConePresent { bool cone_present = 1; }
+
+message McuShutdownScheduled {
```

Comment:
we can remove the `Mcu` at the beginning, all messages here are coming from the Mcu because it is an `McuToJetson` message